products/public-websites/README.md

```diff
@@ -511,10 +512,10 @@ VA.gov header and footer, within VA.gov context, including presentation of banne
 * URL: [https://www.va.gov/](https://www.va.gov/) 
 
 **More info:**
-* 2019 research for launch: [https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/global/header-footer-and-navigation](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/global/header-footer-and-navigation)
+* 2019 research for launch: [https://github.com/department-of-veterans-affairs/va.gov-team/blob/fb49a95fea9d755534cb7f6a77060e221625d5a1/products/information-architecture/archive/2019-2020-navigation-docs/primary-header-footer/2019-header-nav-outline.md](https://github.com/department-of-veterans-affairs/va.gov-team/blob/fb49a95fea9d755534cb7f6a77060e221625d5a1/products/information-architecture/archive/2019-2020-navigation-docs/primary-header-footer/2019-header-nav-outline.md)
```

Comment:
Was a dead link. Is this the correct new link?

Reply:
Let's remove this. Dave has overhauled the existing docs, and I dont' think we need this pointer anymore.

Reply:
👍